packages/components/src/components/radio-button/radio-button.stories.ts

```diff
@@ -31,142 +32,87 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * Use the `size` attribute to change the size of the input radio. This attribute affects the font-size within the element, while the element itself remains the same size.
+ * Use the `size` attribute to change the size of the input radio.
  */
 
 export const Size = {
-  parameters: { controls: { exclude: ['size'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-radio-button size="lg">
+        <sd-icon library="global-resources" name="system/picture" slot="icon"></sd-icon>
+      </sd-radio-button>
+      <sd-radio-button size="md">
+        <sd-icon library="global-resources" name="system/picture" slot="icon"></sd-icon>
+      </sd-radio-button>
+      <sd-radio-button size="sm">
+        <sd-icon library="global-resources" name="system/picture" slot="icon"></sd-icon>
+      </sd-radio-button>
+    </div>
+  `
 };
 
 /**
- * Use the show-label attribute to show the label of the radio button.
```

Comment:
* Use the `label` attribute to show the label of the radio button.
 
 -> Rename this attribute, pls. Then we are aligned with sd-button.

Reply:
I think this would be a breaking change. What do you think @mariohamann?

Reply:
It would be. OMG. What did we do here? Please make fix ticket to deprecate this strange thing.

Reply:
Created https://github.com/solid-design-system/solid/issues/1307